tests_integration/nativeapp/test_init_run.py

```diff
@@ -492,6 +492,59 @@ def run():
             assert result.exit_code == 0
 
 
+# Tests that application post-deploy scripts are executed even when they are used with --project and project is in another directory
```

Comment:
Is it possible to parametrize the previous test instead using a similar strategy to `is_versioned`?

Reply:
done.